src/auth/Gadgets.cc

```diff
@@ -129,10 +129,6 @@ authenticateCachedUsersList()
 #if HAVE_AUTH_MODULE_NEGOTIATE
     if (Auth::SchemeConfig::Find("negotiate"))
         u1 = Auth::Negotiate::User::Cache()->sortedUsersList();
-#endif
-#if HAVE_AUTH_MODULE_NTLM
-    if (Auth::SchemeConfig::Find("ntlm"))
-        u2 = Auth::Ntlm::User::Cache()->sortedUsersList();
 #endif
     if (u1.size() > 0 || u2.size() > 0) {
```

Comment:
After PR changes, `u2` is guaranteed to be empty here. The whole "paired" merging code in this function looks odd to me -- why not merge each container right after it is computed? -- but I do not insist on rewriting it. However, if you do not want to rewrite this code, please at least add a comment to note that `u2` is always empty here. For example,

```suggestion
    // TODO: Rewrite to avoid an always empty u2.
    if (u1.size() > 0 || u2.size() > 0) {
```